pandas/tests/resample/test_resample_api.py

```diff
@@ -771,3 +771,58 @@ def test_end_and_end_day_origin(
     )
 
     tm.assert_series_equal(res, expected)
+
+
+@pytest.mark.parametrize("numeric_only", [True, False])
```

Comment:
Can you also add lib.no_default

Reply:
Sure, I will add `lib.no_default` to my code.